dist/video/ds4/video.css

```diff
@@ -56,3 +56,6 @@
 .video-player--poster .shaka-scrim-container {
   visibility: hidden;
 }
+.shaka-controls-button-panel .report-button > svg {
```

Comment:
Is `report-button` their class or ours? If ours it should conform to BEM, e.g. (`video-player__report-btn`)

Reply:
May as well also add support for RTL.

Reply:
its ours, but it's not a direct descendant of video-player at all... would it be better to have something like 
`.shaka-controls-button-panel__report-btn`? Although shaka-controls-button-panel isn't ours

Reply:
Can you paste the full markup in the PR so I can get a better understanding.

Reply:
![Screen Shot 2021-07-20 at 4 05 10 PM](https://user-images.githubusercontent.com/25092249/126406207-40408a54-b805-429f-8dc4-b609d6c4380a.png)

Reply:
It's a descendant of our `video-player` base class, so the double-underscore BEM syntax is appropriate. i.e. `video-player__report-button`